packages/shared/src/components/auth/RegistrationForm.tsx

```diff
@@ -84,7 +84,9 @@ export const RegistrationForm = ({
         ref={formRef}
         onSubmit={onSubmit}
       >
-        {socialAccount && <ImageInput initialValue={socialAccount.image} />}
+        {socialAccount && (
```

Comment:
We need to ensure that this is non-editable for the social flow.
See WT-348, (just a reminder if you can pick that one up after this?)

Reply:
Would be more than happy to take it 🚢